src/python/pants/subsystem/subsystem.py

```diff
@@ -1,15 +1,20 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+import importlib
 import inspect
-from typing import Dict, Optional, Tuple, Type, TypeVar, Union, cast
+import logging
+from typing import Callable, Dict, Iterable, List, Optional, Tuple, Type, TypeVar, Union, cast
 
 from pants.option.optionable import Optionable
 from pants.option.options import Options
 from pants.option.scope import ScopeInfo
 from pants.subsystem.subsystem_client_mixin import SubsystemClientMixin, SubsystemDependency
 
 
+logger = logging.getLogger(__name__)
```

Comment:
@cosmicexplorer @blorente what should this be? I think we don't want to use the logging library directly anymore, but don't remember why or what to do instead? Something about Rust doing the logging?

Reply:
This is actually correct! As of #6817, this will log within rust! See https://github.com/pantsbuild/pants/blob/a731138eb4f360e59d3c0de4e39dc168d573887b/src/rust/engine/logging/src/logger.rs#L74!

Reply:
Yeah, this will get magically and invisibly piped to rust, and will use the unified logging mechanism we have :)